examples/run/sycl/seq_example_sycl.sycl

```diff
@@ -69,11 +68,11 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
             const traccc::common_options& common_opts, bool run_cpu) {
 
     // Read the surface transforms
-    auto surface_transforms = traccc::read_geometry(i_cfg.detector_file);
+    auto surface_transforms = traccc::io::read_geometry(i_cfg.detector_file);
 
     // Read the digitization configuration file
     auto digi_cfg =
-        traccc::read_digitization_config(i_cfg.digitization_config_file);
+        traccc::io::read_digitization_config(i_cfg.digitization_config_file);
```

Comment:
Personally I find the use of `auto` in the IO code a bit excessive. I know it was already like this so I'm not saying we should change it in this PR per sé, but at some point I think we should explicitly use our nice typedefs.

Reply:
Yeah... I agree. Maybe in a follow-up PR.